webpack.config.devtools.js

```diff
@@ -8,17 +9,28 @@ const ignoreRegexes = [
   /chrome-remote-debug-protocol/
 ];
 
+const nativeMapping = {
+  "public/js/utils/source-editor": "devtools/client/sourceeditor/editor"
+};
+
 module.exports = webpackConfig => {
   webpackConfig.output.library = "Debugger";
   webpackConfig.externals = [
     function(context, request, callback) {
+      const mod = path.join(context.replace(__dirname + "/", ""), request);
+
       // Any matching paths here won't be included in the bundle.
       if(ignoreRegexes.some(r => r.test(request))) {
         callback(null, "var {}");
         return;
+      } else if(nativeMapping[mod]) {
+        callback(null, "var devtoolsRequire('" + nativeMapping[mod] + "')");
```

Comment:
This is the magic that hooks into the local modules when in the panel. `devtoolsRequire` is the privileged require that will be available in the panel context. It will generate code like `var editor = devtoolsRequire("devtools/editor")`.